streamwebs_frontend/streamwebs/forms.py

```diff
@@ -34,6 +34,20 @@ class UserProfileForm(forms.ModelForm):
     class Meta:
         model = UserProfile
         fields = ('school', 'birthdate')
+
+
+class MacroinvertebratesForm(forms.ModelForm):
+    class Meta:
+        model = Macroinvertebrates
+        fields = ('school', 'date_time', 'weather', 'site', 'time_spent',
+                  'num_people', 'riffle', 'pool', 'caddisfly', 'mayfly',
+                  'riffle_beetle', 'stonefly', 'water_penny', 'dobsonfly',
+                  'clam_or_mussel', 'crane_fly', 'crayfish',
+                  'damselfly', 'dragonfly', 'scud', 'fishfly', 'alderfly',
+                  'mite', 'aquatic_worm', 'blackfly', 'leech', 'midge',
+                  'snail', 'mosquito_larva', 'wq_rating',
+                  'somewhat_sensitive_total', 'sensitive_total',
+                  'tolerant_total')
         labels = {
             'school': _('School'),
             'birthdate': _('Birthdate'),
```

Comment:
You no longer need labels, since we included verbose_names in the models form fields